### DISCUSSION TITLE
let users change the login page background with a image

### DISCUSSION THREAD

**Discussion by Sparklingx** [authelia/authelia#5169] [Ideas]
Example:

like this: 
note:( I used photo editor to made this to demonstrate) 

![image](https://user-images.githubusercontent.com/85538352/229773152-01ac1999-94a8-4577-bb89-1b5ec716c0a8.png)

**Comment by vincejv** [comment]
is this already possible without compiling from source?

**Comment by Sparklingx** [comment]
no
